src/userInterface/panels/CommandPanel/CommandPanel.tsx

```diff
@@ -267,7 +250,7 @@ export const CommandPanel: React.FC<{
             // updated version
             editedConfig: JsonConfigCommandCenter.currentJson,
             // Latest update
-            originalConfig: JsonConfigCommandCenter.getOriginalJsonConfig(),
+            originalConfig: props.originalJsonConfig,
```

Comment:
This should be props.originalJsonConfig?.data

Reply:
done